lib/output/output.go

```diff
@@ -1,11 +1,8 @@
 package output
 
-import (
-	"encoding/json"
-	"fmt"
-	"sort"
-	"strings"
-)
+type OutputBuilder interface {
```

Comment:
ah, so we *are* declaring a public interface here... This is intended to communicate to implementors of new output types that their output struct, say `xmlOutput`, should minimally receive a `GenerateOutput` method, right?